### PR TITLE
LPS-156315 Provide default value to object fields

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/object/object-api/src/main/resources/com/liferay/object/service/packageinfo [hunk 1]
```diff
@@ -1 +1 @@
-version 19.0.0
\ No newline at end of file
+version 20.0.0
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-api/src/main/resources/com/liferay/object/service/packageinfo:1]: Hi @seltong,

Move this change to the proper commit,

Thank you!

File: modules/apps/object/object-admin-rest-api/src/main/resources/com/liferay/object/admin/rest/dto/v1_0/util/packageinfo [hunk 2]
```diff
@@ -1 +1 @@
-version 2.0.0
\ No newline at end of file
+version 2.1.0
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-admin-rest-api/src/main/resources/com/liferay/object/admin/rest/dto/v1_0/util/packageinfo:1]: Hi @seltong,

Move this change to the semver commit.

Thanks!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Very nice @seltong!

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
Resent here: #224

**Pull request closed without merging**